src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -180,6 +182,25 @@ const AuthenticatorButton: UISchemaElementComponent<{
             {description}
           </Typography>
         )}
+        {nickname && (
+          <Typography
+            paragraph
+            id={`${iconName}-nickname`}
+            sx={{
+              fontSize: '.875rem',
+              margin: 0,
+              marginBlockEnd: '6px',
+              textAlign: 'start',
+              overflow: 'hidden',
+              whiteSpace: 'nowrap',
+              textOverflow: 'ellipsis',
+            }}
+            data-se="authenticator-button-nickname"
+            className={classNames('authenticator-enrollment-nickname', 'no-translate')}
+          >
+            {nickname}
+          </Typography>
```

Comment:
Can you attach screenshots of how this looks? Would be helpful to get some idea of what all these styles add up to. Additionally, might be good to update the UI Demo mock to show nickname in one of the authenticator buttons.

Reply:
Attached screenshots, however, can you give more info re: the ui demo mock? I have seen it in some screenshots but not sure how you guys get that to display? Do you know what mock is used and do you just update the responseConfig file?

Reply:
I think he's talking about `_ui-demo.json` which you can add to the `responseConfig`
See: https://okta.slack.com/archives/C04NYNLLRDL/p1698082416073859

Reply:
Ohh, awesome! Thank you! I will get that updated 🙏